create_nf_samplesheet.sh

```diff
@@ -28,9 +30,18 @@ mkdir -p ${FQDIR}
 
 find "${DATADIR}" -name "*.fastq.gz" -exec ln -s {} "${FQDIR}/" \;
 
-python "$(find "$ENVPATH/sw/rnaseq" -name "fastq_dir_to_samplesheet.py" -print -quit)" \
-  --sanitise_name \
-  --sanitise_name_index=1 \
-  --strandedness=reverse \
-  "${FQDIR}" \
-  "${PROJDIR}/${PROJ}.SampleSheet.csv"
+
+if [[ $PIPELINE == "rnaseq" || $PIPELINE == "methylseq" ]]
```

Comment:
Is it possible to do `$PIPELINE != "sarek"` instead?

Alternatively, there could be an ELSE clause where the scripts exits with "Pipeline not supported"